docs/edge-authorizers/discovery-flow.mdx

```diff
@@ -128,8 +127,8 @@ The expected response should be:
             "max_delay_seconds": 120,
             "min_delay_seconds": 60
           },
-          "resource": "<POLICY_ID>/bundle.tar.gz",
-          "service": "aserto-bundler",
+          "resource": "<TENANT>/<POLICY>/blobs/sha256:<DIGEST>",
+          "service": "<UUID>-tenant",
```

Comment:
should this be `"service": "<tenant-id>"` ?

Reply:
Actually for the service identification we use the registry connection id and the registry org